zds/tutorialv2/models/__init__.py

```diff
@@ -2,11 +2,65 @@
 
 from django.utils.translation import ugettext_lazy as _
 
+
+CONTENT_TYPES = (
+    # Configuration of content names
+    # name                : the ID of content (unique and uppercase)
+    # verbose_name        : name of content
+    # verbose_name_plural : plural name of content
+    # parent_name         : the name of category who contain this content
+    # validation_before   : True or False ; True in content must be validate before publication (and if JsFiddle is
+    #                       available)
+    # single_container    : True or False ; True if the content is a single container
+    # beta                : True or False ; True if the content can be in beta
+    {
+        'name': 'TUTORIAL',
+        'verbose_name': 'tutoriel',
+        'verbose_name_plural': 'tutoriels',
+        'parent_name': 'tutoriel',
+        'validation_before': True,
+        'single_container': False,
+        'beta': True,
+    },
+    {
+        'name': 'ARTICLE',
+        'verbose_name': 'article',
+        'verbose_name_plural': 'articles',
+        'parent_name': 'article',
+        'validation_before': True,
+        'single_container': True,
+        'beta': True,
+    },
+    {
+        'name': 'OPINION',
+        'verbose_name': 'billet',
+        'verbose_name_plural': 'billets',
+        'parent_name': 'tribune',
+        'validation_before': False,
+        'single_container': True,
+        'beta': False,
+    },
+)
+
+# a list of contents they need a validation before publication
+CONTENT_TYPES_VALIDATION_BEFORE = [content['name'] for content in CONTENT_TYPES if content['validation_before']]
+
+# a list of contents they only have one big container which contain small containers
```

Comment:
"a list of contents which have one big container containing at least one small container"